### PR TITLE
feat: add help screen

### PR DESCRIPTION
**Pull request by SSelinn**
## Description

This branch adds a help canvas that shows the keybinds and their actions. These are not changeable.

## Testability

You can go to the scene _Settings_ where you can from the menu go to settings and then to keybinds and back. This also works with F1 and within the game.

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch
- [ ] Added a scene to the game for your changes

### CODE REVIEW

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs [hunk 1]
```diff
@@ -144,60 +126,82 @@ public void OnPlayerDeath(HealthComponent _)
...
     /// <summary>
     /// This button toggles the menu canvas off and the setting canvas on or game settings on and off.
     /// </summary>
     public void OnToggleSettings()
     {
-        if (_preventSettingsToggle) return;
+        _isOnSettings = !_isOnSettings;
 
-        if (_currentSceneName == _sceneNameStartingMenu) // Toggle from menu to menu settings and back.
+        if (_currentSceneName == _sceneNameStartingMenu)
```
Comment (SilasPeters) [inline aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:149]: You could rewrite this if else if else if statement using a `switch` statement. On other places as well

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs [hunk 2]
```diff
@@ -144,60 +126,82 @@ public void OnPlayerDeath(HealthComponent _)
...
+            SettingCanvas.SetActive(!_isOnHelp);
+        }
+        else if (_currentSceneName == _sceneNameGame) // Toggle from game to helpscreen and back.
+        {
+            if (_isOnHelp)
+            {
+                GameManager.Instance.Pause();
+                MenuOpenedEvent?.Invoke(_isOnHelp);
+            }
+            else if (!_isOnHelp && !_isOnSettings)
```
Comment (SilasPeters) [inline aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:182]: the `!_isOnHelp` is trivially true, for this branch of the if else statement is not run unless true.

**Review by SilasPeters — CHANGES_REQUESTED** [review]
Looking good, but not good

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs (hunk 2 above)
Comment (SSelinn) [reply aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:182]: That is indeed true! Fixed it.

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs (hunk 1 above)
Comment (SSelinn) [reply aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:149]: Done this as well!

File: aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs [hunk 3]
```diff
@@ -9,63 +9,58 @@
...
-    /// </summary>
-    private bool _isOnGameSettings;
-
-    /// <summary>
-    /// To ensure the menu settings and menu UI aren't on at the same time.
+    /// Name of the start screen.
     /// </summary>
-    private bool _isOnMenuSettings;
+    [SerializeField]
+    private string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
```
[the remark this answers is not in the document]
Comment (SSelinn) [reply aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs:57]: Once the PR in is main than we can assign the right scene to it.

**Pull request closed without merging**